js/src/tooltip.js

```diff
@@ -37,8 +37,7 @@ import BaseComponent from './base-component'
 const NAME = 'tooltip'
 const DATA_KEY = 'bs.tooltip'
 const EVENT_KEY = `.${DATA_KEY}`
-const CLASS_PREFIX = 'bs-tooltip'
-const BSCLS_PREFIX_REGEX = new RegExp(`(^|\\s)${CLASS_PREFIX}\\S+`, 'g')
+const CLASS_PREFIX = 'bs-tooltip-auto'
```

Comment:
This class (`bs-tooltip-auto`) can be changed to just `tooltip` :thinking:

Reply:
for now was renamed to ` bs-tooltip ` and you 'll find it in line 72. It can be omitted with the proper changes on scss